modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/search/util/DDMSearchHelper.java

```diff
@@ -157,6 +172,19 @@ public SearchContext buildTemplateSearchContext(
 		searchContext.setAttribute("language", language);
 		searchContext.setAttribute("mode", mode);
 		searchContext.setAttribute("resourceClassNameId", resourceClassNameId);
```

Comment:
@rodrigopaulino should we remove this from the search context? we are not indexing this attribute.

Reply:
Yes, we do index it for DDMTemplate types.